### ISSUE TITLE
Listes de thèmes/projets

### ISSUE THREAD

**Issue by degehi** [degehi/GeFoPro-www#3]
Dans toutes l'application, proposer les thèmes les plus utilisés en premier, pour une sélection plus aisée.

Note: la problématique n'apparait pas avec un fonctionnement de thèmes associés à une année d'apprentissage car le tri s'effectue déjà de cette manière et donc la liste est courte. De plus, le dernier thème utilisé dans les journaux de travail est déjà proposé par défaut. Ne faut-il pas revoir la structure métier des thèmes avant de développer cette fonctionnalité?

**Comment by cererinoros** [comment]
> Ne faut-il pas revoir la structure métier des thèmes avant de développer cette fonctionnalité?

Oui, je suis d'accord avec cette remarque. J'estime que moins 20 % des thèmes sont utilisés régulièrement dans notre section AUT.

**Comment by degehi** [comment]
Remarque concernant le tri de la structure des thèmes: étant donné que les thèmes peuvent être dorénavant gérés pas l'application, il est très facile de repérer ceux qui n'ont jamais été utilisé dans les journaux de travail des apprentis car l'icône d'effacement apparait pour ceux-ci!
